common/jsonquery/converters.go

```diff
@@ -66,3 +66,14 @@ func (convertor) ObjectToMap(node *ajson.Node) (map[string]any, error) {
 
 	return result, nil
 }
+
+// NodeToArray treats current node as an array of objects.
+// Sometimes JSON response from the server right of the bet is an array. This method performs the necessary conversion.
+func (convertor) NodeToArray(node *ajson.Node) ([]*ajson.Node, error) {
```

Comment:
This is a small utility function. 
When getting `{baseURL}/oauth/token/accessible-resources` the response is array `[]` instead of `{}`.
Thats the primary reason for this method. Existing methods to get array expect the key, while in this case it is "root", "dot".